### DISCUSSION TITLE
How to use this extension?

### DISCUSSION THREAD

**Discussion by bvqbao** [fwouts/previewjs#406] [Q&A] [answered]
Hi,

I install the extension on vscode and try to preview a very simple component in vuejs. It doesn't work and I don't know why.
I'm not sure if I use it correctly. I try to read the docs but no help.

<img width="767" alt="image" src="https://user-images.githubusercontent.com/4357219/161223358-73c00eed-5afa-427d-8975-e90711535cd3.png">

So how can I preview a component?

Thanks,

Edited: it didn't work even with a hello world component:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/4357219/161237203-a4397da6-9144-4192-9b09-21d1c41ac73d.png">

**Comment by fwouts** [comment, marked as answer]
Hey @bvqbao, thanks for this, it looks like you found a serious bug!

Would you be able to [file an issue](https://github.com/fwouts/previewjs/issues/new?assignees=&labels=pending+triage%2Cvscode&template=bug_report_vscode.yml) so I have all the info I need to debug it further?

Ideally if you can also share your `package.json` (or better yet a public repo) that'd be super helpful.
